clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -306,29 +330,80 @@ const PrivacyRequestForm: React.FC<PrivacyRequestFormProps> = ({
+                isRequired={identityInputs.name === "required"}
               >
-                <FormLabel fontSize="sm">
-                  {typeof item === "string"
-                    ? key[0].toUpperCase() + key.slice(1)
-                    : item.label}
-                </FormLabel>
-                {key === "phone" ? (
-                  <PhoneInput
-                    id={key}
-                    name={key}
-                    onChange={(value) => {
-                      setFieldValue(key, value, true);
-                    }}
-                    onBlur={handleBlur}
-                    value={values[key]}
-                  />
-                ) : (
+                <FormLabel fontSize="sm">Name</FormLabel>
+                <Input
+                  id="name"
+                  name="name"
+                  focusBorderColor="primary.500"
+                  placeholder="Michael Brown"
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  value={values.name}
+                />
+                <FormErrorMessage>{errors.name}</FormErrorMessage>
+              </FormControl>
+            ) : null}
+            {identityInputs.email ? (
+              <FormControl
+                id="email"
+                isInvalid={touched.email && Boolean(errors.email)}
+                isRequired={identityInputs.email === "required"}
+              >
+                <FormLabel fontSize="sm">Email</FormLabel>
+                <Input
+                  id="email"
+                  name="email"
+                  type="email"
+                  focusBorderColor="primary.500"
+                  placeholder="your-email@example.com"
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  value={values.email}
+                />
+                <FormErrorMessage>{errors.email}</FormErrorMessage>
+              </FormControl>
+            ) : null}
+            {identityInputs.phone ? (
+              <FormControl
+                id="phone"
+                isInvalid={touched.phone && Boolean(errors.phone)}
+                isRequired={identityInputs.phone === "required"}
+              >
+                <FormLabel fontSize="sm">Phone</FormLabel>
+                <PhoneInput
+                  id="phone"
+                  name="phone"
+                  onChange={(value) => {
+                    setFieldValue("phone", value, true);
+                  }}
+                  onBlur={handleBlur}
+                  value={values.phone}
+                />
+                <FormErrorMessage>{errors.phone}</FormErrorMessage>
+              </FormControl>
+            ) : null}
+            {Object.entries(identityInputs)
+              .filter(
+                ([key, item]) =>
+                  key !== "email" &&
+                  key !== "phone" &&
+                  key !== "name" &&
+                  typeof item !== "string"
+              )
```

Comment:
Same filtering to render the form fields

Reply:
nit: You'll find this kind of rendering logic easier to manage if you pre-filter this above, like
```
const defaultIdentityInputs = Object.entries(identityInputs).filter(...)
const customIdentityInputs = Object.entries(identityInputs).filter(...)
```

And then you can use those variables directly with less fuss 😄